### ISSUE TITLE
Rename _circa columns to _est

### ISSUE THREAD

**Issue by davidmcclure** [sul-cidr/kb-api#11]

**Comment by kgeographer** [comment]
year_abt are years given by Nick as being 'about' or 'around'

year_est are calculated values, where no date existed at all. we computed an estimated year from family lifespans (forget particulars, code is around) and the mean error was 4 years +/- - not great but it was decided to include them at the end of coalesce statements (and edit those that led to people living over 110 years!)

one of the items in the backlog of "refresh" items is making the existing interface (KB1) reflect the uncertainty of [actual/about/estimated]. I don't think we need to worry about that viz task at this time, but the model should retain the difference, in case...maybe collapsing all years and adding a boolean birth_est along with the birth_abt ??

also, the 'year_est_pass' field was supposed to record what pass the estimated year was from -- that is, after each estimation pass generated years, a subsequent pass used the newly generated years to try again on those that had failed for insufficient primary data.

I can see that 20 rows have values that are years, but that is not the original intent of the field and I doubt it is used. It should have had integers between 1 and 9, I believe.

**Comment by davidmcclure** [comment]
Hey @kgeographer,

Cool, thanks for this. Even beyond the `_est` and `_abt` fields, the event dates are tricky because we've also got a mix of precision - some rows have a year/month/day "date" field, others just have years. And, the years have the varying level of precision.

Here's the current logic in the migration:
- If `event_date` is defined, write the date into the `date` field on the new model.
- If the date is empty, try to find a year, in this order of precedence: `year` -> `year_abt` -> `year_est`. Write this value into the new `year` column.
- If a year is found from either the `year_abt` or `year_est` field, flip on a boolean field called `year_est`, which records the fact that the year is approximate.

Here's the code: https://github.com/sul-cidr/kb-api/blob/master/lib/import/event_dates.rb. The main question I've got is - in the new model, do we need to distinguish between years derived from the old `_est` vs. `_abt` columns? I think we definitely need to keep track of the fact that a year is "circa," in effect, which is what the new `year_est` boolean flag does.

But, does it matter how the value was derived? My guess is that in the UI we'll end up using the same indicator either way - something like "~1820", or "circa 1820."
